cmd/find-main.go

```diff
@@ -142,7 +142,7 @@ EXAMPLES:
        $ {{.HelpName}} s3 --larger 64MB --smaller 1GB --print {url}
 
    08. Find all objects created in the last week under "s3/bucket".
-       $ {{.HelpName}} s3/bucket --newer 1w
+       $ {{.HelpName}} s3/bucket --newer-than 7d
```

Comment:
Is there any reason why we can't use `1w` like the text above describes?

Reply:
Find uses 
``` 
var unitMap = map[string]int64{
	"ns": int64(Nanosecond),
	"us": int64(Microsecond),
	"ms": int64(Millisecond),
	"s":  int64(Second),
	"m":  int64(Minute),
	"h":  int64(Hour),
	"d":  int64(Day),
} 
``` 
Thus week is not supported.